examples/couchbase-ami/README.md

```diff
@@ -23,7 +23,7 @@ To build the Couchbase AMI:
 1. Configure your AWS credentials using one of the [options supported by the AWS 
    SDK](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html). Usually, the easiest option is to
    set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables.
-1. Update the `variables` section of the `couchbase.json` Packer template to specify the AWS region and Couchbase
+1. Update the `variables` section of the `couchbase.json` to specify the AWS region and Couchbase
```

Comment:
Why remove the text Packer template?

Reply:
I see it now - I think the capitilization of "Packer" was throwing me off. I'll remove that change.